odoo/tools/misc.py

```diff
@@ -1380,6 +1380,26 @@ def _format_time_ago(env, time_delta, lang_code=False, add_direction=True):
     return babel.dates.format_timedelta(-time_delta, add_direction=add_direction, locale=locale)
 
 
+def format_number(number, decimal=1):
```

Comment:
format_decimalized_number ? (https://en.wikipedia.org/wiki/Decimalisation -> using powers of 10, seems somewhat related to what we do even if we are not converting pennies into dollars)

cc @mart-e still strugling with a name, help is welcomed :)

Reply:
@tde-banana-odoo 
@mart-e 

It's not exactly a "decimalization" because we use power of 10^3 and not power of 10

Technically, it's an engineer notation
https://en.wikipedia.org/wiki/Engineering_notation

Reply:
Well I let you choose, anyway something better than "format_number" that tells nothing. Decimalization is probably something that you would understand when reading it.

format_engineering_number is ok for me, a bit verbose but at least it is correct.